### ISSUE TITLE
Multi Select Input Fields - Is it possible to keep the placeholder there after you select, and before you begin typing?

### ISSUE THREAD

**Issue by austintruchan** [matrix-msu/Kora3#89]

**Comment by Dev34** [comment]
Yes

**Comment by austintruchan** [comment]
lol

**Comment by Dev34** [comment]
Added placeholders that stay after you click/focus on the input. Only added to searchable inputs.

Options for reviewing: New Field w/ Field Type as Documents or Video (searchable multi-select input is in Advanced Options). Should work for all searchable inputs though.

FYI: Issue outlined in #46 causes the placeholders to be cropped. This is because the width of the input is constantly being changed at runtime and is too small to accommodate the placeholder.

This is on the multi-select-placeholder-#89 branch.

**Comment by austintruchan** [comment]
@Dev34 Okay. This seems good other than the fact the placeholder text gets cropped! Should we leave that to kevin to fix in #46 or is that something you should fix here for this instance?

**Comment by Dev34** [comment]
I think Kevin has figured it out. Once his changes are pulled I'll make sure my fix still works and pass it off for review

**Comment by Dev34** [comment]
Commits from #46 did not end up fixing the width overrides of inputs at runtime, so the placeholders were still getting cropped. I discovered this is being caused by some sort of jquery maintenance function and this isn't invoked by any Kora code. I've added a work-around into jquery that stops this from occurring so now the placeholders should remain after the input is clicked on and they should not be cropped.

This is on the multi-select-placeholder-#89 branch (pushed to GitHub)
